systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlIsolatedST.java

```diff
@@ -0,0 +1,106 @@
+import io.strimzi.test.WaitException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static io.strimzi.systemtest.Constants.CRUISE_CONTROL;
+import static io.strimzi.systemtest.Constants.REGRESSION;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@Tag(REGRESSION)
+@Tag(CRUISE_CONTROL)
+public class CruiseControlIsolatedST extends BaseST {
+
+    private static final Logger LOGGER = LogManager.getLogger(CruiseControlIsolatedST.class);
+    private static final String NAMESPACE = "cruise-control-isolated-test";
+
+    private static final String CRUISE_CONTROL_METRICS_TOPIC = "strimzi.cruisecontrol.metrics"; // partitions 1 , rf - 1
+    private static final String CRUISE_CONTROL_MODEL_TRAINING_SAMPLES_TOPIC = "strimzi.cruisecontrol.modeltrainingsamples"; // partitions 32 , rf - 2
+    private static final String CRUISE_CONTROL_PARTITION_METRICS_SAMPLES_TOPIC = "strimzi.cruisecontrol.partitionmetricsamples"; // partitions 32 , rf - 2
+
+
+    @Test
+    void testManuallyCreateMetricsReporterTopic() {
+        KafkaResource.kafkaWithCruiseControlWithoutWaitAutoCreateTopicsDisable(CLUSTER_NAME, 3, 3);
+
+        PodUtils.verifyThatRunningPodsAreStable(KafkaResources.zookeeperStatefulSetName(CLUSTER_NAME));
+        PodUtils.verifyThatRunningPodsAreStable(KafkaResources.kafkaStatefulSetName(CLUSTER_NAME));
+
+        PodUtils.waitUntilPodIsInCrashLoopBackOff(PodUtils.getFirstPodNameContaining("cruise-control"));
+
+        LOGGER.info("Verifying that samples topics are not present because of " +
+            "'Cruise Control cannot find partitions for the metrics reporter that topic matches strimzi.cruisecontrol.metrics in the target cluster'");
+
+        assertThrows(WaitException.class, CruiseControlUtils::verifyThatCruiseControlSamplesTopicsArePresent);
+
+        LOGGER.info("Verifying that metrics reporter topic is not present because of selected config 'auto.create.topics.enable=false'");
+
+        assertThrows(WaitException.class, CruiseControlUtils::verifyThatKafkaCruiseControlMetricReporterTopicIsPresent);
+
+        LOGGER.info("Creating samples topics {},{}", CRUISE_CONTROL_MODEL_TRAINING_SAMPLES_TOPIC, CRUISE_CONTROL_PARTITION_METRICS_SAMPLES_TOPIC);
+
+        KafkaTopicResource.topic(CLUSTER_NAME, CRUISE_CONTROL_MODEL_TRAINING_SAMPLES_TOPIC)
+            .editSpec()
+                .withPartitions(32)
+                .withReplicas(2)
+                .addToConfig("cleanup.policy", "delete")
+            .endSpec()
+            .done();
+
+        KafkaTopicResource.topic(CLUSTER_NAME, CRUISE_CONTROL_PARTITION_METRICS_SAMPLES_TOPIC)
+            .editSpec()
+                .withPartitions(32)
+                .withReplicas(2)
+                .addToConfig("cleanup.policy", "delete")
+            .endSpec()
+            .done();
```

Comment:
you should not create these two topics manually, they are created by CC when it detects that the `CRUISE_CONTROL_METRICS_TOPIC` is created and can continue.